src/pubd/repository.rs

```diff
@@ -1311,11 +1328,14 @@ impl RepoStats {
 
 impl fmt::Display for RepoStats {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        writeln!(f, "RSYNC BASE:        {}", self.sia_base)?;
```

Comment:
Why "RSYNC" and not "SIA" at in the stats endpoint?

Reply:
see other comment